keyboards/satan/keymaps/admiralStrokers/keymap.c

```diff
@@ -147,7 +149,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 				XXXX, XXXX, XXXX, 					XXXX,			  		XXXX, XXXX, XXXX, XXXX),
 };
 
-const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) { // MACRODOWN only works in this function
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {3 // MACRODOWN only works in this function
```

Comment:
That is the weird thing here, I have the last version compiled here. And the keyboard was working as expected, even with the capitalized "T" in checktime? I have no idea how that worked out.. I'm typing on the keyboard with the keymap I initially pushed.

Is there a way to run travis prior to syncing?